src/fiat/drhook/internal/dr_hook_procinfo.F90

```diff
@@ -8,15 +8,15 @@
 ! nor does it submit to any jurisdiction.
 !
 
-SUBROUTINE DR_HOOK_PROCINFO(KMYPROC, KNPROC)
+SUBROUTINE DR_HOOK_PROCINFO(KMYPROC, KNPROC, LMPI_INITIALIZED)
 USE EC_PARKIND  ,ONLY : JPIM
 USE MPL_MPIF
 IMPLICIT NONE
-INTEGER(KIND=JPIM),INTENT(OUT) :: KMYPROC, KNPROC
-LOGICAL :: LMPI_INITIALIZED
```

Comment:
LMPI_INITIALIZED should probably be also a INTEGER(KIND=JPIM) as it is accessed as such from C. Fortran usually uses 32 bit for a logical, but it is safer to not make this assumption. Alternatively a new wrapper function `c_dr_hook_procinfo BIND(C)` could be wrapping this existing one and convert to integer there?

Reply:
Ahh, good catch! I tried changing it to `INTEGER(KIND=JPIM)` and there was no errors or warnings. Adding a wrapper feels a bit unnecessary for something so small?